src/cli.ts

```diff
@@ -28,6 +28,24 @@ Publish package versions from one registry to another.`)
     boolean: true,
     default: false
   })
+  .option('latest-only', {
+    demand: false,
+    describe: 'Only syncs the latest dist-tag version',
+    boolean: true,
+    default: false
+  })
+  .option('latest-majors', {
+    demand: false,
+    describe: 'Only syncs the latest majors',
```

Comment:
Codacy has a fix for the issue: [Replace `····describe:·'Only·syncs·the·latest·majors'` with `······describe:·"Only·syncs·the·latest·majors"`](https://app.codacy.com/gh/joebowbeer/regsync/pullRequest?prid=9263163)
```suggestion
      describe: "Only syncs the latest majors",
```